content/api/commands/intercept.md

```diff
@@ -1537,6 +1537,50 @@ are some workarounds:
   entirely. For more information, see
   [this comment on issue #14459](https://github.com/cypress-io/cypress/issues/14459#issuecomment-768616195)
 
+## Command Log
+
+```javascript
+cy.intercept('/accounts*').as('accountsGet')
+cy.intercept('/company', { companyId: 1 }).as('companyGet')
+cy.intercept('/teams*', [{ teamId: 2 }]).as('teamsGet')
+```
+
+Whenever you create `cy.intercept()` rules, Cypress will display a new
+Instrument Log called _Routes_. It will list the routing table in the Instrument
```

Comment:
Elsewhere (see [The Test Runner](https://deploy-preview-4061--cypress-docs.netlify.app/guides/core-concepts/test-runner#Instrument-Panel) and [session](https://deploy-preview-4061--cypress-docs.netlify.app/api/commands/session#The-Instrument-Panel)) we called these Instrument Panels. Can we change that name throughout this PR?

Reply:
copied the terminology from cy.route's docs, i'll update it in both places

Reply:
de87058